<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1158</width>
    <height>668</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Whatever Name</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="1">
     <widget class="QTabWidget" name="tabWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>500</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>515</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>data control</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>fit control</string>
       </attribute>
       <widget class="QPushButton" name="bManFit">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>180</y>
          <width>81</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Manual Fit</string>
        </property>
       </widget>
       <widget class="QSlider" name="sKnotsNum">
        <property name="geometry">
         <rect>
          <x>119</x>
          <y>140</y>
          <width>131</width>
          <height>20</height>
         </rect>
        </property>
        <property name="minimum">
         <number>5</number>
        </property>
        <property name="maximum">
         <number>9</number>
        </property>
        <property name="pageStep">
         <number>1</number>
        </property>
        <property name="value">
         <number>5</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QLabel" name="lKnotsNum">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>140</y>
          <width>101</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Number of knots:</string>
        </property>
       </widget>
       <widget class="QLabel" name="lShowKnotsNum">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>120</y>
          <width>68</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>5</string>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QComboBox" name="cbFunctionSelect">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>180</y>
          <width>91</width>
          <height>21</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>tanh_multi</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>tanh_0out</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>tsplfun</string>
         </property>
        </item>
       </widget>
      </widget>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QFrame" name="frame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>634</width>
        <height>0</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0" colspan="9">
        <widget class="MplCanvasWrapper" name="mplCanvas" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="3">
        <widget class="QLabel" name="lDataDescription">
         <property name="text">
          <string>Data Description:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="8">
        <widget class="QComboBox" name="cRhoPsi">
         <item>
          <property name="text">
           <string>rho</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>psi</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="lEfit">
         <property name="text">
          <string>Efit:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="lShot2">
         <property name="text">
          <string>Shot:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2" colspan="2">
        <widget class="QLabel" name="lShowShot">
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::Panel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <property name="text">
          <string>100000</string>
         </property>
        </widget>
       </item>
       <item row="2" column="4">
        <widget class="QLabel" name="lTime2">
         <property name="text">
          <string>Time(msec):</string>
         </property>
        </widget>
       </item>
       <item row="2" column="5">
        <widget class="QLabel" name="lShowTime">
         <property name="minimumSize">
          <size>
           <width>46</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::Panel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <property name="text">
          <string>3000</string>
         </property>
        </widget>
       </item>
       <item row="2" column="6" colspan="3">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>324</width>
           <height>14</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="lShot1">
         <property name="text">
          <string>Shot:</string>
         </property>
         <property name="buddy">
          <cstring>sShot</cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="1" colspan="2">
        <widget class="QSpinBox" name="sShot">
         <property name="maximum">
          <number>100000</number>
         </property>
         <property name="value">
          <number>100000</number>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QLabel" name="lTime1">
         <property name="text">
          <string>Time:</string>
         </property>
         <property name="buddy">
          <cstring>sTime</cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="4">
        <widget class="QSpinBox" name="sTime">
         <property name="maximum">
          <number>100000</number>
         </property>
         <property name="value">
          <number>3000</number>
         </property>
        </widget>
       </item>
       <item row="3" column="5" colspan="2">
        <widget class="QPushButton" name="bPlot">
         <property name="text">
          <string>Plot</string>
         </property>
        </widget>
       </item>
       <item row="3" column="7">
        <spacer name="horizontalSpacer_1">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>65</width>
           <height>22</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="8">
        <widget class="QPushButton" name="bSave">
         <property name="text">
          <string>Save</string>
         </property>
        </widget>
       </item>
      </layout>
      <zorder>mplCanvas</zorder>
      <zorder>lDataDescription</zorder>
      <zorder></zorder>
      <zorder>horizontalSpacer</zorder>
      <zorder></zorder>
      <zorder>cRhoPsi</zorder>
      <zorder>horizontalSpacer_1</zorder>
      <zorder>bSave</zorder>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MplCanvasWrapper</class>
   <extends>QWidget</extends>
   <header>mplCanvasWrapper.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>sKnotsNum</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lShowKnotsNum</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>824</x>
     <y>179</y>
    </hint>
    <hint type="destinationlabel">
     <x>835</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
